runtime/README.md

```diff
@@ -601,6 +601,59 @@ Table: `SHUTDOWN` output arguments
 | chksum       | u32       | Checksum over other output arguments, computed by Caliptra. Little endian.
 | fips_status  | u32       | Indicates if the command is FIPS approved or an error
 
+### ADD\_SUBJECT\_ALT\_NAME
```

Comment:
Now that we're adding more commands, maybe we should mark the spec version as 1.1 and bump the internal version number.

The 1.0 spec will be published as a PDF, so I think it's fine for main to always be tracking the latest changes.

cc @FerralCoder @nquarton I'm not sure if you folks have already thought about this for the LMS commands.

Reply:
Done

Reply:
This is a good point. The PR that adds LMS to this doc (https://github.com/chipsalliance/caliptra-sw/pull/1411) is still out, so I can just rebase it on this when it goes in. I'll comment to make sure it's not missed.